src/lib/routes/admin-api/event.ts

```diff
@@ -152,7 +187,9 @@ export default class EventController extends Controller {
         res: Response<FeatureEventsSchema>,
     ): Promise<void> {
         const feature = req.params.featureName;
-        const eventList = await this.eventService.searchEvents({ feature });
+        const eventList = await this.eventService.deprecatedSearchEvents({
```

Comment:
Are we planning on updating these method calls too or would that be too much work?